packages/frameworks/test/frameworks.unit.test.ts

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+import { existsSync } from 'fs';
+import { Framework } from '../';
+
+function isString(arg: any): arg is string {
+  return typeof arg === 'string';
+}
+
+describe('frameworks', () => {
+  it('ensure there is an example for every framework', async () => {
+    const root = path.join(__dirname, '..', '..', '..');
+    const getExample = (name: string) => path.join(root, 'examples', name);
+
+    const frameworks = require('../frameworks.json') as Framework[];
+
+    const result = frameworks
+      .map(f => f.slug)
```

Comment:
Are there some frameworks which don't have a slug? And why not?

Reply:
@styfle the `Other` framework has `null` as its slug.